packages/qwik-city/buildtime/markdown/mdx.ts

```diff
@@ -20,16 +20,38 @@ export async function createMdxTransformer(ctx: BuildContext): Promise<MdxTransf
   const userRemarkPlugins = userMdxOpts.remarkPlugins || [];
   const userRehypePlugins = userMdxOpts.rehypePlugins || [];
 
+  const coreMdxPlugins = ctx.opts.mdxPlugins;
+
+  const coreRemarkPlugins = [];
+
+  if (typeof coreMdxPlugins?.remarkGfm === 'undefined' || coreMdxPlugins.remarkGfm) {
+    coreRemarkPlugins.push(remarkGfm);
+  }
+
+  const coreRehypePlugins = [];
+
+  if (typeof coreMdxPlugins?.rehypeSyntaxHighlight === 'undefined' || coreMdxPlugins.rehypeSyntaxHighlight) {
+    coreRehypePlugins.push(rehypeSyntaxHighlight);
+  }
+
+  if (typeof coreMdxPlugins?.rehypeAutolinkHeadings === 'undefined' || coreMdxPlugins.rehypeAutolinkHeadings) {
+    coreRehypePlugins.push(rehypeAutolinkHeadings);
+  }
```

Comment:
If configuration is missing (`undefined`) or it's set to `true`, the core plugins are added (no breaking change).